openshift/backup.bc.yaml

```diff
@@ -0,0 +1,86 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: "${NAME}-build-template"
+  creationTimestamp: 
+parameters:
+- name: NAME
+  displayName: Name
+  description: The name assigned to all of the resources defined in this template.
+  required: true
+  value: backup
+- name: GIT_REPO_URL
+  displayName: Git Repo URL
+  description: The URL to your GIT repo.
+  required: true
+  value: https://github.com/BCDevOps/backup-container.git
+- name: GIT_REF
+  displayName: Git Reference
+  description: The git reference or branch.
+  required: true
+  value: master
```

Comment:
we should use a commit here instead of master.

Reply:
Maybe!  I don't mind receiving new updates on master and can always fail back.